themes/harper/static/css/main.css

```
@@ -0,0 +1,459 @@
+.more__date {
+  font-size: 13px;
+  color: var(--mut);
+  width: 80px;
+  flex: none;
+  font-variant-numeric: tabular-nums;
+}
+.more__title { font-size: 17px; font-weight: 500; }
+.more__row:hover .more__title { color: var(--acc); }
+
+/* ---------- Notes feed ---------- */
+.notes { }
```

Stylelint (17.14.0)

[error] Empty block (block-no-empty)

(block-no-empty)